sdk/src/connections/usb/windows/usb_depth_sensor_windows.cpp

```diff
@@ -1270,18 +1270,14 @@ aditof::Status UsbDepthSensor::initTargetDepthCompute(uint8_t *iniFile,
     return Status::OK;
 }
 
-aditof::Status
-UsbDepthSensor::getIniParams(std::map<std::string, float> &params) {
-    using namespace aditof;
-    Status status = Status::UNAVAILABLE;
-    return status;
+aditof::Status UsbDepthSensor::getDepthComputeParams(
+    std::map<std::string, std::string> &params) {
+    return aditof::Status::OK;
 }
 
-aditof::Status
-UsbDepthSensor::setIniParams(const std::map<std::string, float> &params) {
-    using namespace aditof;
-    Status status = Status::UNAVAILABLE;
-    return status;
+aditof::Status UsbDepthSensor::setDepthComputeParams(
+    const std::map<std::string, std::string> &params) {
+    return aditof::Status::OK;
```

Comment:
What is the reason for switching to OK instead of keeping the UNVAIALABLE?

Reply:
Changed back to Status::UNAVAILABLE